mesonbuild/modules/python.py

```diff
@@ -441,11 +458,16 @@ def _get_win_pythonpath(name_or_path: str) -> T.Optional[str]:
             return None
 
     def _find_installation_impl(self, state: 'ModuleState', display_name: str, name_or_path: str, required: bool) -> MaybePythonProg:
+        build_config = (
+            self.interpreter.environment.properties[MachineChoice.BUILD].get_python_build_config()
+            or self.interpreter.environment.coredata.get_option(OptionKey('python.build_config'))
+        )
```

Comment:
This feels a bit like we should be `setdefault`ing it in `Environment.__init__` (we do this already for properties that can also be set from environment variables, e.g. `BOOST_ROOT`, but not currently for options). But I'm not sure. Just a note for later, for now...

Reply:
I looked a bit further into the machine property files and it seems that the `python_build_config` property is unnecessary. This can already be set in the `[built-in options]` section.

```ini
[built-in options]
python.build_config = '...'
```

Reply:
Ah yes, excellent point. Nice to be able to drop that as unnecessary. :)